example_decompme_mappings.toml

```diff
@@ -0,0 +1,8 @@
+# Decompme Mappings
+
+[decompme_compiler_identifier]
```

Comment:
I think this settings section could/should just go into permuter_settings.toml unless you think there's a good reason to keep it separate

Reply:
permuter_settings.toml only exists in repos, but we don't have a repo at the point we do `./import.py https://decomp.me/...`.